build.gradle

```diff
@@ -77,6 +77,10 @@ allprojects { subproj ->
 	apply plugin: 'com.github.ben-manes.versions' // gradle dependencyUpdates
 	apply from: "$rootDir/gradle/degraph.gradle"
 
+	tasks.withType(Test) { task ->
+		task.testLogging.exceptionFormat = 'full'
+	}
+
```

Comment:
FYI: the `junitPlatformTest` task created by the Gradle plugin is **not** a `Test` task but rather a `JavaExec` task.

So I don't see how changing the logging for `Test` tasks will help you here.